vmaas/reposcan/database/csaf_store.py

```diff
@@ -46,7 +48,129 @@ def _save_csaf_files(self, csaf_files: CsafFiles) -> list[int]:
         self.conn.commit()
         return db_ids
 
+    def _get_product_rows(self, csaf_cves):
```

Comment:
I understand that you want to have the same signature for `_get_product_rows` and `_get_cve_rows` so you're returning but I think it would be better to use OOP for this, create a base class (as an interface in different OOP languages) with `get_rows` method raises just NotImplementedError and implement that method in classes inheriting from base class. It would improve readability and mainly maintainability.